first-app/src/components/list/List.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import useStyles from './style';
+import List from '@material-ui/core/List';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import arrLists from './arrLists';
+
+
+export default function MainContentList() {
+
+    const classes = useStyles();
+
+    return(
+        <>
+            <List 
+                component="nav"
+                aria-label="main mailbox folders"
+                subheader={
+                    <ListSubheader component="div" disableGutters='true' className={classes.listHeader}>
```

Comment:
if you set some prop to the component and doesn't give some value it becomes true